CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-authn-token-http ChangeLog
 
+## 8.1.0 - 2023-06-08
+
+### Fixed
+- Uniform HTTP errors with invalid username hitting `/hash-parameters`.
+  Return fake hash parameters in case of invalid username.
```

Comment:
```suggestion
- Return uniform errors from `/authn/tokens/<token-type>/hash-parameters`.
- Return fake hash parameters in case of invalid account information.

### Added
- Introduce `hmacKey` in options to be used when generating fake tokens.
- Introduce `jitter` in options to obfuscate timing.

```